src/tests/modify.test.js

```diff
@@ -397,3 +397,118 @@ describe('modify() - basic mutations', () => {
     });
   });
 });
+
+describe('modify() - create new fields', () => {
+  it('create base field', () => {
+    const result = modify(schemaAddress, {
+      create: {
```

Comment:
I've rebased this (Create PR) with main https://github.com/remoteoss/json-schema-form/pull/83, including the new specs discussed in the previous threads. Ready for a last review :)

Reply:
(FYI: I have a call now, will be back in 1h at ~15 UTC.) Then I'll finish updating the last PR #79

Reply:
Changes look good, just have a question but not a blocker! (I'll be out between ~15-17UTC, leaving this approved!)

Reply:
Thanks! I removed the last piece that was not needed. @joaoalmeida-remote could you re-approve?

Reply:
With the #78 merged, I rebased this PR with main to solve conflicts, and deleted the last test (it was skipped) as I realized we didn't need it anyway. [a33dd02](https://github.com/remoteoss/json-schema-form/pull/80/commits/a33dd02dbdc79ea0e8ee87698fcfa11a1bc71b48)